Doc/library/getpass.rst

```diff
@@ -16,7 +16,7 @@
 
 The :mod:`getpass` module provides two functions:
 
-.. function:: getpass(prompt='Password: ', stream=None)
+.. function:: getpass(prompt='Password: ', stream=None, *, echochar=None)
```

Comment:
Let's add an underscore throughout for readability and accessibility:

```suggestion
.. function:: getpass(prompt='Password: ', stream=None, *, echo_char=None)
```

Reply:
thank you. left `echochar` only in function names for tests

Reply:
Let's update those too: we need to read them as well, and if we search for the name, they will also show up.

Reply:
done